plugin.audio.radiothek/addon.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.audio.radiothek" name="ORF Radiothek" version="0.0.1" provider-name="sofaking">
+    <requires>
+        <import addon="xbmc.python" version="2.26.0" />
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="default.py">
+        <provides>audio</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en">ORF Radiothek - Radio Streaming</summary>
+        <summary lang="de">ORF Radiothek - Radio Streaming</summary>
+        <description lang="en">This is a Kodi Addon to provides access to the ORF Radiothek (Austrian public service broadcaster) Streaming Portal (Live Radio, Podcasts, Archive, ...). </description>
+        <description lang="de">Die ORF Radiothek stellt das Radiomaterial der öffentlich-rechtlichen Radiosender aus Österreich zur Verfügung. Neben Livestreaming können auch vergangene Sendungen und eine Vielzahl an Podcasts abgerufen werden.</description>
+        <language>de en</language>
+        <license>GNU General Public License, v2</license>
```

Comment:
can you use a spdx license identifier here? `GPL-2.0-only`
see: https://spdx.org/licenses/